command/command_test.go

```diff
@@ -122,11 +122,11 @@ func testVaultServerWithKVVersion(tb testing.TB, kvVersion string) (*api.Client,
 func testVaultServerAllBackends(tb testing.TB) (*api.Client, func()) {
 	tb.Helper()
 
+	handlers := newVaultHandlers()
 	client, _, closer := testVaultServerCoreConfig(tb, &vault.CoreConfig{
-		CredentialBackends: credentialBackends,
-		AuditBackends:      auditBackends,
-		LogicalBackends:    logicalBackends,
-		BuiltinRegistry:    builtinplugins.Registry,
```

Comment:
Was dropping initializing BuiltinRegistry on purpose here?

Reply:
Oops. Reverted the drop in [c293ac0](https://github.com/hashicorp/vault/pull/27515/commits/c293ac0eb382666c16170377890ab8d37e8311af)